src/axom/inlet/examples/example1_expected_documentation.rst

```diff
@@ -0,0 +1,106 @@
+==================

```

Comment:
Here's the expected documentation output file.

Reply:
Did you run this through sphinx to see what it looked like?

Reply:
Sphinx doesn't have an easy one command (at least that i could find) that will render the page for you.  I did find this that will show you basically what it will look like.

http://rst.ninjs.org/

Reply:
and here I was copy-pasting into a dummy git repo all this time